react-time-track/src/services/user.js

```diff
@@ -57,4 +58,41 @@ async function getUserProjects(user_id) {
   return response.json();
 }
 
-export { userList, createUser, getUserProjects };
+async function getUser(userId) {
+  console.log(userId);
```

Comment:
Hi, don't forget to delete this console-logs

Reply:
Thanks for notice them. I'll replace it right now.